torch/onnx/symbolic_opset13.py

```diff
@@ -4,9 +4,10 @@
 # This file exports ONNX ops for opset 13
 import torch
 import torch.onnx.symbolic_helper as sym_help
-from torch.onnx.symbolic_helper import parse_args, _unimplemented
-from torch.onnx.symbolic_opset9 import overload_by_arg_count, _maybe_cast_reduce_op_input, nonzero, expand
-from torch.onnx.symbolic_opset11 import unsqueeze
+from torch.onnx.symbolic_helper import _squeeze_helper, parse_args, _unimplemented
```

Comment:
please address flake8 issues

Reply:
And you may wish to use this to run flake8 locally when you commit:
https://github.com/pytorch/pytorch/blob/master/tools/git-pre-commit

You can add it to your local git hooks:
`ln -s ../../tools/git-pre-commit .git/hooks/pre-commit`